nio_command_block.py

```diff
@@ -19,26 +19,25 @@ class SecurityMethod(Enum):
 
 
 class URLParameter(PropertyHolder):
-    prop_name = ExpressionProperty(title="Property Name")
-    prop_value = ExpressionProperty(title="Property Value")
+    prop_name = Property(title="Property Name")
+    prop_value = Property(title="Property Value")
 
 
 class BasicAuthCreds(PropertyHolder):
-    username = StringProperty(title='Username')
-    password = StringProperty(title='Password')
+    username = StringProperty(title='Username', default='Admin')
+    password = StringProperty(title='Password', default='Admin')
```

Comment:
I know we use this all the time, but I don't think it's appropriate to include these hardcoded in the block.